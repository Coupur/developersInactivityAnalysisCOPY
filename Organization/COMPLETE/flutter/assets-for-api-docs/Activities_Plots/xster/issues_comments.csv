id,date,creator_login
415598632,2018-08-23 23:00:04,xster
415609012,2018-08-23 23:56:09,xster
416025816,2018-08-26 09:33:15,xster
