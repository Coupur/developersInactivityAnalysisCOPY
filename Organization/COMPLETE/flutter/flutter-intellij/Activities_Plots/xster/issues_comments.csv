id,date,creator_login
500974179,2019-06-11 18:48:13,xster
501510843,2019-06-13 01:13:25,xster
430407170,2018-10-16 21:27:38,xster
430410101,2018-10-16 21:37:35,xster
427963000,2018-10-08 20:08:44,xster
428252668,2018-10-09 16:07:25,xster
416769037,2018-08-28 23:06:46,xster
414853774,2018-08-21 23:25:16,xster
413694109,2018-08-16 21:44:05,xster
413946841,2018-08-17 18:11:39,xster
415576322,2018-08-23 21:26:18,xster
396429322,2018-06-12 00:38:40,xster
413429612,2018-08-16 05:24:45,xster
360612489,2018-01-25 21:51:45,xster
365436040,2018-02-13 23:05:15,xster
365436937,2018-02-13 23:08:55,xster
313249387,2017-07-05 22:58:24,xster
