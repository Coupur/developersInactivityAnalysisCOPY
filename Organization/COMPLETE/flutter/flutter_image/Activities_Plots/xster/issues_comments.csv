id,date,creator_login
374831359,2018-03-21 04:27:17,xster
