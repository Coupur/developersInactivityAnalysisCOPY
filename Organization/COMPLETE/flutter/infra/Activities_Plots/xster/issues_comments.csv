id,date,creator_login
498483975,2019-06-04 01:27:10,xster
