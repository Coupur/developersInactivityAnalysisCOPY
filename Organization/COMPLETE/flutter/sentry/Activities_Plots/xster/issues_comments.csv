id,date,creator_login
375053917,2018-03-21 18:43:16,xster
374768015,2018-03-20 21:46:07,xster
374764943,2018-03-20 21:35:11,xster
322343470,2017-08-15 00:12:37,xster
322340378,2017-08-14 23:50:18,xster
