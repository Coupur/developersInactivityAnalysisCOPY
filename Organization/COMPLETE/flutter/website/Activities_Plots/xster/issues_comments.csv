id,date,creator_login
510233690,2019-07-10 21:21:17,xster
509444785,2019-07-09 00:51:29,xster
506094079,2019-06-27 00:39:31,xster
507483182,2019-07-02 01:33:45,xster
498421328,2019-06-03 20:49:49,xster
484269131,2019-04-17 21:31:07,xster
481965303,2019-04-11 04:57:48,xster
482732679,2019-04-12 21:44:07,xster
484247239,2019-04-17 20:24:22,xster
484253623,2019-04-17 20:42:51,xster
472554510,2019-03-13 18:43:01,xster
469970507,2019-03-06 05:08:32,xster
465368937,2019-02-20 00:31:14,xster
464282125,2019-02-16 03:48:42,xster
457692680,2019-01-25 19:20:47,xster
447183386,2018-12-14 01:39:22,xster
447147022,2018-12-13 22:39:33,xster
500583630,2019-06-10 20:34:25,xster
500689921,2019-06-11 05:16:30,xster
438507262,2018-11-14 01:48:25,xster
436005273,2018-11-05 19:31:59,xster
458805716,2019-01-30 04:15:35,xster
434020471,2018-10-29 18:12:58,xster
424427214,2018-09-25 17:15:04,xster
424441665,2018-09-25 17:58:18,xster
415220826,2018-08-22 23:30:31,xster
412748567,2018-08-14 04:05:48,xster
413003908,2018-08-14 20:24:54,xster
399753561,2018-06-24 12:35:24,xster
389347108,2018-05-15 23:32:51,xster
390065525,2018-05-18 01:24:34,xster
378730308,2018-04-04 20:14:14,xster
378719089,2018-04-04 19:34:11,xster
375128706,2018-03-21 23:26:21,xster
372226755,2018-03-12 08:22:55,xster
351145859,2017-12-12 18:36:25,xster
351219578,2017-12-12 22:42:16,xster
345802705,2017-11-20 19:28:53,xster
346440937,2017-11-22 18:47:50,xster
346164534,2017-11-21 21:20:10,xster
340114086,2017-10-27 22:42:03,xster
321434091,2017-08-10 02:35:44,xster
311746784,2017-06-28 18:28:14,xster
305617025,2017-06-01 20:52:49,xster
288576074,2017-03-22 23:57:38,xster
283495339,2017-03-01 22:42:10,xster
