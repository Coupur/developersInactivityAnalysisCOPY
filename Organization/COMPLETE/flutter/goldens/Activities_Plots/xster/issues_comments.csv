id,date,creator_login
481890501,2019-04-10 22:31:08,xster
